ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```diff
@@ -443,6 +443,16 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-ui
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-efs
+- as: aws-ipi-ovn-stress-f30
+  cron: 17 19 30 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "5"
+    test:
+    - chain: openshift-e2e-test-qe-stresstest
```

Comment:
Does the test support ARM ? If supported, then it's better to move the test to ARM to save cost (as ARM is cheaper than AMD on AWS)

Reply:
@huiran0826 I supposed it supports ARM. But do you have a preferred profile to run this test given it just runs on one profile for now?

Reply:
It supports ARM, but this is the first case that we selected in this stressTest, for a long term, it might include other component cases. And I'm thinking it's better we use a broadly support platform.

Reply:
@liangxia According to Huiran's comment, let's keep it on AWS-OVN. Please give a final review and approve.